projects/v2/src/app/components/report/report.component.ts

```diff
@@ -139,17 +140,18 @@ export class ReportComponent implements OnInit, AfterViewInit {
 
   ngAfterViewInit() { }
 
-  makeOntologyLinksGraphData(reportData: Report,sheetData: Row[]) {
+  makeOntologyLinksGraphData(reportData: Report, sheetData: Row[]) {
     const { result, biomarkersSeperateNames } =
       this.reportService.makeAllOrganReportDataByOrgan(sheetData, this.asFullData);
-    
+
     const biomarkerCols = [];
     biomarkersSeperateNames.forEach((bm) => {
-      if (this.displayedColumns.includes(bm.name) === false){
+      if (this.displayedColumns.includes(bm.name) === false) {
         biomarkerCols.push(bm.name);
-      }    
+      }
       this.displayedColumns = [
         'organName',
+        'tableVersion',
```

Comment:
Table version does not need to be shown in the UI, only in the xlsx export.

Reply:
Column needs to be `table_version` and not `tableVersion`

Reply:
I don't see any table_version string in your code. Can you explain?

Reply:
Also, I made a big oops. It IS actually supposed to be `tableVersion` in the sheets. Not sure if that was a change on the humnanatlas.io site or I just totally flubbed it. My bad. Please change back to `tableVersion`.